__snapshots__/a11y-input.md

```diff
@@ -0,0 +1,10 @@
+# `a11y-input.js`
```

Comment:
1. Snapshots are not generated by the `@open-wc init` command.
1. `@open-wc` Karma Testing ref: https://www.npmjs.com/package/@open-wc/testing-karma
1. Snapshot ref: https://www.npmjs.com/package/karma-snapshot